lib/deterministicmnlist/SimplifiedMNListDiff.js

```diff
@@ -180,8 +180,15 @@ SimplifiedMNListDiff.fromObject = function fromObject(obj, network) {
   if (obj.merkleRootQuorums) {
     simplifiedMNListDiff.merkleRootQuorums = obj.merkleRootQuorums;
   }
-  if (simplifiedMNListDiff.mnList.length >= 1) {
+
+  if (simplifiedMNListDiff.mnList.length > 0) {
+    if (network && simplifiedMNListDiff.mnList[0].network.name !== network.name) {
```

Comment:
Jsdoc says that it can be string, but there's no conversion to object from string in the method. Am I'm missing something?

Reply:
Ah yes I'm missing what's hidden :D